skills.tex

```diff
@@ -1,5 +1,5 @@
 \section{SKILLS}
 \begin{bulletlist}
-   \item{Languages: Bash, C, C\#, CSS, HTML, Java, MySQL, Node, Python, React, Redux, Ruby, TypeScript}
-   \item{Tools \& Practices: AWS, Docker, DynamoDB, Elasticsearch, Git, GitLab CI, Jira, NPM, Scrum, Terraform}
+   \item{Languages: C\#, Go, Java, Kotlin, Node, React, Redux, Rust, SQL, Swift, SwiftUI, Terraform, TypeScript}
```

Comment:
Typescript?!?!? 🤧 
For real though, put this with Node/React. You want those terms to be all together since they're similar.

Reply:
Ah so with this I have the lists in alphabetical order since I that made the most sense. Do you think I should deviate from that?

Also thinking about removing typescript - it's straightforward and should probably be inferred with Node/React/Redux on here. Though maybe I want it to hit those keywords?? What do you think?

Reply:
I prefer the alphabetical - keep it simple (better for scanning too)

Reply:
Going to resolve this and leave it alphabetical (and leave typescript in)